Formula/scala.rb

```diff
@@ -17,17 +17,17 @@ class Scala < Formula
   option "with-docs", "Also install library documentation"
   option "with-src", "Also install sources for IDE support"
 
-  depends_on :java => "1.8+"
+  depends_on java: "1.8+"
```

Comment:
Strange. Possibly a `.rubocop.yml` file not being loaded locally? I'll see if I can reproduce the problem later.